src/main/java/org/snomed/snowstorm/core/data/domain/Concept.java

```diff
@@ -29,6 +29,7 @@
 	public interface Fields extends SnomedComponent.Fields {
 		String CONCEPT_ID = "conceptId";
 		String DEFINITION_STATUS_ID = "definitionStatusId";
+		String PATH = "path";
```

Comment:
This could go in the `SnomedComponent` class.

Reply:
Good idea.